### PR TITLE
Codacywarnings

### PR DESCRIPTION
**Pull request by lordfolken**
# The purpose of this change

# The source of the data (for e.g. new frequencies)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-   1. map bounding box co-ordinates.
+1.   `content`: The content itself, like:
+    -   Waypoints
+    -   Airspaces
+
+2.   `remote`: URLs to external content, like:
+    -   Waypoints hosted on https://soaringweb.org/, etc.
+
+3.   `source`: Configuration parameters specifying how to generate content, like:
+    -   map bounding box co-ordinates
```
[the remark this answers is not in the document]
Comment (rawtaz) [reply README.md:19]: Also, change the last line to "Map bounding box coordinates".

File: README.md [hunk 2]
```diff
@@ -33,20 +34,17 @@ E.g. `ca_quebec.*` implies Canada.
...
-1. https://download.xcsoar.org/repository (XCSoar)
-1. https://download.xcsoar.org/waypoints/waypoints.js (website)
-1. https://download.xcsoar.org/waypoints/waypoints_compact.js (website)
-1. https://download.xcsoar.org/maps/maps.config.js (website & `mapgen`)
-1. https://download.xcsoar.org/waypoints/xcsoar_waypoints.cup (`mapgen`)
+1.   [https://download.xcsoar.org/repository](https://download.xcsoar.org/repository) (XCSoar)
+2.   [https://download.xcsoar.org/waypoints/waypoints.js](https://download.xcsoar.org/waypoints/waypoints.js) (website)
+3.   [https://download.xcsoar.org/waypoints/waypoints_compact.js](https://download.xcsoar.org/waypoints/waypoints_compact.js) (website)
+4.   [https://download.xcsoar.org/maps/maps.config.js](https://download.xcsoar.org/maps/maps.config.js) (website & `mapgen`)
+5.   [https://download.xcsoar.org/waypoints/xcsoar_waypoints.cup](https://download.xcsoar.org/waypoints/xcsoar_waypoints.cup) (`mapgen`)
```
Comment (rawtaz) [inline README.md:45]: Is there a reason to be redundant with the URL here? If we ditch the `[]()` I believe the URLs will still be links in the end, at least this is the case with https://soaringweb.org/ above.

Comment (rawtaz) [reply README.md:45]: Also, the bullet points are indented one space too much.

File: README.md [hunk 3]
```diff
@@ -55,9 +53,10 @@ When doing so, please write in the comments the source of the new  data, so that
 
 Contributions here have to pass:
 
-1. A review by another contributer
-2. A parsing check by [aerofiles](https://github.com/Turbo87/aerofiles)
-3. Validity check depending on file type. E.g. remote URLs must exist, file extensions must be correct, etc.
+1.   A review by another contributer
+2.   A parsing check by [aerofiles](https://github.com/Turbo87/aerofiles)
+3.   Validity check depending on file type. E.g. remote URLs must exist, file extensions must be correct, etc.
```
Comment (rawtaz) [inline README.md:58]: Bullet points indented one space too much.

File: README.md [hunk 4]
```diff
@@ -55,9 +53,10 @@ When doing so, please write in the comments the source of the new  data, so that
 
 Contributions here have to pass:
 
-1. A review by another contributer
-2. A parsing check by [aerofiles](https://github.com/Turbo87/aerofiles)
-3. Validity check depending on file type. E.g. remote URLs must exist, file extensions must be correct, etc.
+1.   A review by another contributer
```
Comment (rawtaz) [inline README.md:56]: s/contributer/contributor/

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -3,13 +3,15 @@ Thank you very much for contributing! Please fill out the following
 questions to make it easier for us to review your changes.
 -->
 
-# The purpose of this change
+# Change Title
```
Comment (rawtaz) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: If it's possible to tell the validator to ignore this file or even better the entire `.github/` directory, I think we should do that. If that's not an option, I think we should s/Change Title/Pull Request/.

**Review by rawtaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (lordfolken) [reply README.md]: I just wanted it to work with any markdown parser. 
but i found a better [official way](https://www.markdownguide.org/basic-syntax/#formatting-links) here.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lordfolken) [reply README.md]: I'm fedup with this check. I do proper syntax and remove the check.

File: README.md [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
+    - Waypoints
+    - Airspaces
+
 2. `remote`: URLs to external content, like:
-   1. Waypoints hosted on https://soaringweb.org/, etc.
+    -  Waypoints hosted on https://soaringweb.org/, etc.
+
 3. `source`: Configuration parameters specifying how to generate content, like:
-   1. map bounding box co-ordinates.
+    -  map bounding box coordinates
```
Comment (rawtaz) [inline README.md:19]: Missing a capital M unless you really want to keep it lowercase :P

File: README.md [hunk 7]
```diff
@@ -1,21 +1,22 @@
...
 
 1. `content`: The content itself, like:
-   1. Waypoints
-   2. Airspaces
+    - Waypoints
+    - Airspaces
+
 2. `remote`: URLs to external content, like:
-   1. Waypoints hosted on https://soaringweb.org/, etc.
+    -  Waypoints hosted on https://soaringweb.org/, etc.
```
Comment (rawtaz) [inline README.md:16]: Is this URL supposed to be in `<>` as well considering you put the ones below in these tags?

**Review by rawtaz — CHANGES_REQUESTED** [review]

**Review by rawtaz — APPROVED** [review]
Let's goo!

**Pull request merged**